EventFilter/L1TRawToDigi/plugins/implementations_stage2/GMTCollections.cc

```diff
@@ -10,11 +10,28 @@ namespace l1t {
          event_.put(std::move(regionalMuonCandsOMTF_), "OMTF");
          event_.put(std::move(regionalMuonCandsEMTF_), "EMTF");
          event_.put(std::move(muons_), "Muon");
+         event_.put(std::move(muonsSet2_), "MuonSet2");
+         event_.put(std::move(muonsSet3_), "MuonSet3");
+         event_.put(std::move(muonsSet4_), "MuonSet4");
+         event_.put(std::move(muonsSet5_), "MuonSet5");
+         event_.put(std::move(muonsSet6_), "MuonSet6");
          event_.put(std::move(imdMuonsBMTF_), "imdMuonsBMTF");
          event_.put(std::move(imdMuonsEMTFNeg_), "imdMuonsEMTFNeg");
          event_.put(std::move(imdMuonsEMTFPos_), "imdMuonsEMTFPos");
          event_.put(std::move(imdMuonsOMTFNeg_), "imdMuonsOMTFNeg");
          event_.put(std::move(imdMuonsOMTFPos_), "imdMuonsOMTFPos");
       }
+
+      MuonBxCollection*
+      GMTCollections::getMuons(const unsigned int set)
+      {
+         if (set == 1) return muons_.get();
```

Comment:
There are arguments for both options. The slots in the uGT crate are numbered from 1 to 12, the uGT boards from 0 to 5. Board 0 is in slot 1 and so forth.